.github/workflows/phpunit_sw_old_versions.yml

```diff
@@ -0,0 +1,88 @@
+# Shopware 5.2 - 5.5 run on PHP versions 5.6, 7.0 and 7.1.
+name: PHPUnit (SW 5.2 - 5.5)
+
+on:
+  push:
+    branches:
+      - main
+      - develop
+  pull_request:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-versions: ['5.6', '7.0', '7.1']
+        shopware-versions: ['5.2', 'v5.2.0', '5.3', 'v5.3.0', '5.4', '5.5']
```

Comment:
Why do we test 5.2. and 5.3. twice?

Reply:
Because 5.2 = 5.2.27 and 5.3 = 5.3.6. The other two versions are explicitly pinned, so ensure that our plugin also works with these .0 versions, as we had issues with them im the past.

Reply:
Ok just saw we did the same in Travis, resolved.